### PR TITLE
fix: revert `AirOptical` refractive index

### PR DESCRIPTION
**Pull request by c-dilks**
### Briefly, what does this PR introduce?
Revert `AirOptical` `RINDEX` change from https://github.com/eic/epic/pull/394/files

Fix https://github.com/eic/epic/issues/430

EDIT: Also condenses some constant material property tables to endpoints only, namely those of the DRICH and PFRICH.

### What kind of change does this PR introduce?
- [x] Bug fix (issue #__)
- [ ] New feature (issue #__)
- [ ] Documentation update
- [ ] Other: __

### Please check if this PR fulfills the following:
- [ ] Tests for the changes have been added
- [ ] Documentation has been added / updated
- [ ] Changes have been communicated to collaborators

### Does this PR introduce breaking changes? What changes might users need to make to their code?
no

### Does this PR change default behavior?
yes, reversion

### CODE REVIEW

File: compact/optical_materials.xml [hunk 1]
```diff
@@ -8,42 +8,42 @@
...
+      3.0996*eV   1.00029
+      3.17908*eV  1.00029
+      3.26274*eV  1.00029
+      3.35092*eV  1.00029
+      3.44401*eV  1.00029
+      3.54241*eV  1.00029
+      3.64659*eV  1.00029
+      3.7571*eV   1.00029
+      3.87451*eV  1.00029
+      3.99949*eV  1.00029
```
Comment (veprbl) [inline compact/optical_materials.xml:45]: My understanding, is that we don't need anything but the endpoints.
```suggestion
```

**Review by veprbl — APPROVED** [review]

File: compact/optical_materials.xml (hunk 1 above)
Comment (c-dilks) [reply compact/optical_materials.xml:45]: Yes, that's right. I will condense ABSLENGTH too.

Comment (c-dilks) [reply compact/optical_materials.xml:45]: (PFRICH and DRICH are guilty of doing this too, but we plan to parameterize those soon anyway)

Comment (c-dilks) [reply compact/optical_materials.xml]: but that is not a good reason to not clean them up too... done!

**Review by niwgit — APPROVED** [review]
The changes seem fine to me. I tested Cherenkov photon propagation in hpDIRC with the changes and it's working.

**Pull request merged**